project/interface.go

```diff
@@ -37,6 +37,7 @@ type APIProject interface {
 	CreateService(name string) (Service, error)
 	AddConfig(name string, config *config.ServiceConfig) error
 	Load(bytes []byte) error
+	ListStoppedContainers(ctx context.Context, services ...string) ([]string, error)
```

Comment:
My point is more, if we had something about `Container` in the `Project` interface, I would prefer something a little bit more *abstract* or *useful* than just having the `StoppedContainer`. Maybe the `Container` term is too technical though but it's only returning a list of ids — it doesn't feel too tied to the `docker` package to me :stuck_out_tongue_closed_eyes:.

```go
type ContainerFilter struct {
    state string
}

func (p *Project) Containers(ctx context.Context, filters ContainerFilters) {
   // […]
}
```